src/test/java/eu/dissco/orchestration/backend/testutils/TestUtils.java

```diff
@@ -24,13 +27,53 @@ public class TestUtils {
   public static final String OBJECT_DESCRIPTION = "Source system for the DWCA of the Tunicate specimen";
   public static final Instant CREATED = Instant.parse("2022-11-01T09:59:24.00Z");
   public static final ObjectMapper MAPPER = new ObjectMapper().findAndRegisterModules();
-  public static final String SANDBOX_URI = "https://sandbox.dissco.tech/";
+  public static final String SANDBOX_URI = "https://sandbox.dissco.tech/orchestrator";
```

Comment:
Think this is an issue as sourcesystem has /orchestrator/source-system but mapping is just /mapping

Reply:
You're right, they should both be `/orchestrator`. Fixed